shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -1035,33 +1052,35 @@ applyRUpd ::
   RewardUpdate (Crypto era) ->
   EpochState era ->
   EpochState era
-applyRUpd ru (EpochState as ss ls pr pp _nm) = EpochState as' ss ls' pr pp nm'
-  where
-    utxoState_ = _utxoState ls
-    delegState = _delegationState ls
-    dState = _dstate delegState
-    (regRU, unregRU) =
-      Map.partitionWithKey
-        (\k _ -> eval (k ∈ dom (_rewards dState)))
-        (aggregateRewards pr $ rs ru)
-    as' =
-      as
-        { _treasury = (addDeltaCoin (_treasury as) (deltaT ru)) <> fold unregRU,
-          _reserves = addDeltaCoin (_reserves as) (deltaR ru)
-        }
-    ls' =
-      ls
-        { _utxoState =
-            utxoState_ {_fees = _fees utxoState_ `addDeltaCoin` deltaF ru},
-          _delegationState =
-            delegState
-              { _dstate =
-                  dState
-                    { _rewards = eval (_rewards dState ∪+ regRU)
-                    }
-              }
-        }
-    nm' = nonMyopic ru
+applyRUpd -- pulsestate
```

Comment:
Could probably drop this change, but it's fine either way.
```suggestion
applyRUpd
```

Reply:
I left it he way it was.